modules/videoio/src/cap_v4l.cpp

```diff
@@ -769,7 +770,7 @@ bool CvCaptureCAM_V4L::open(const char* _deviceName)
     frame_allocated = false;
     deviceName = _deviceName;
     returnFrame = true;
-    compatibilityMode = utils::getConfigurationParameterBool("OPENCV_VIDEOIO_V4L_RANGE_NORMALIZED", true);
+    compatibilityMode = !utils::getConfigurationParameterBool("OPENCV_VIDEOIO_V4L_RANGE_NORMALIZED", false);
```

Comment:
this was correct before: 
```
compatibilityMode <=>  "converted to/from [0,1) range" <=> OPENCV_VIDEOIO_V4L_RANGE_NORMALIZED <=> 0/1
```

maybe either rename either `compatibilityMode` or `OPENCV_VIDEOIO_V4L_RANGE_NORMALIZED` to match each other.

Reply:
Thanks!
Renamed to `normalizePropRange`.